src/cmdhandler/command_handler.cpp

```diff
@@ -151,7 +151,7 @@ std::string command_handler::state_system(const std::vector<std::string>& tokens
             (*state_to_save.mutable_collectors())[collector.first] = collector.second;
         }
 
-        std::fstream output(save_name, std::ios::out | std::ios::trunc | std::ios::binary);
+        std::fstream output("states/"+save_name, std::ios::out | std::ios::trunc | std::ios::binary);
```

Comment:
It would be good to have `"states/"` be a private constant so that we can change the directory easier in the future.
Like `const std::string SAVE_STATE_DIR  = "states/"`